site/cypress/e2e/Cypress-Gradeable/late_submission_warning_messages.spec.js

```diff
@@ -118,66 +129,89 @@ const SubmitAndCheckMessage = (gradeable_type, upload_file1, invalid_late_day, v
     cy.logout();
 };
 
-it('Checks whether daylight savings warning message should be appearing at current date.', () => {
-    /*
-    First we set the date to a date close to the spring daylight savings time and then check that the message appears.
-    Then we change the date to a month later, and check that the "daylight" message doesn't appear.
-    */
-    const currentDate = new Date();
-    const futureDate = new Date(currentDate);
-    futureDate.setDate(currentDate.getDate() + 7);
+const checkDaylightBanner = (late_days, existence) => {
+    cy.login('instructor');
```

Comment:
I don't think that you should have to log out and log back in each time you run this, you could just login before all of them, and then let the global after each logout at the end